client/src/components/login/ProfilePage.jsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+import axios from "axios";
+
+const ProfilePage = () => {
+  const { user, isAuthenticated } = useAuth0();
+
+    const storeUserInfo = async () => {
+      try {
+        if (user) {
+          const userData = {
+            name: user.given_name || user.name || "DefaultName",
+            email: user.email || "DefaultEmail@example.com",
+            picture: user.picture || "default-profile-picture-url",
+            sub: user.sub,
+          };
+
+          const response = await axios.post(
+            "http://localhost:5005/storeUserInfo",
```

Comment:
Suggestion: this is a more general point for the project but I would spend some time looking at how we can avoid adding the full domain every time we call our backend. In my phone will find a link later. This way you can change domain in one place rather than multiple times throughout project.

Reply:
noted .. and I do change it to the URL variable